src/test/java/herokuapp/CheckboxesTests.java

```diff
@@ -0,0 +1,44 @@
+package herokuapp;
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+
+import java.util.concurrent.TimeUnit;
+
+public class CheckboxesTests {
+    private WebDriver driver;
+
+    @BeforeClass
+    public void setUp() {
+        WebDriverManager.chromedriver().setup();
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.manage().timeouts().implicitlyWait(5, TimeUnit.SECONDS);
+    }
+
+    @AfterClass
+    public void tearDown() {
+        driver.quit();
+    }
+
+    @Test
+    public void checkboxTest() {
+
+        driver.get("http://the-internet.herokuapp.com/checkboxes");
+        WebElement firstCheckbox = driver.findElement(By.cssSelector("[type=checkbox 1]"));
```

Comment:
This is incorrect locator, and it doesn't work. 

Please use driver.findElements() to  get a List of checkboxes and then access them by index e.g. checkboxes.get(0/1);